pkg/tsdb/cloudwatch/log_sync_query.go

```diff
@@ -47,7 +49,17 @@ var executeSyncLogQuery = func(ctx context.Context, e *cloudWatchExecutor, req *
 			return nil, err
 		}
 
+		refId := "A"
+		if q.RefID != "" {
+			refId = q.RefID
+		}
+
 		getQueryResultsOutput, err := e.syncQuery(ctx, logsClient, q, logsQuery, instance.Settings.LogsTimeout.Duration)
+		var sourceError errorsource.Error
+		if errors.As(err, &sourceError) {
+			resp.Responses[refId] = errorsource.Response(sourceError)
+			continue
```

Comment:
This looks like a change in behavior, since sometime we'll have a `SourceError` here and sometimes we won't - is that intentional? If so, what's the motivation?

Reply:
If we'd rather I can change it back to a return. My thought process was that usually when we do response errors we still try running the other queries in the batch.

Reply:
Makes sense! I'm good with leaving it.